src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -57,131 +64,162 @@ public static SidecarInfo createSidecarInfo(TestInputs input) {
   @TestFactory
   Stream<DynamicTest> testCombinations() {
 
+
     List<TestInputs> inputs = List.of(
         // Linux
         new TestInputs(
             "Linux OS info with VS Code",
-            "Linux",
-            "22.0413",
+            "linux",
+            "aarch64",
+            "2.1.2",
             OperatingSystemType.Linux,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            null
+            null,
+            "Confluent-for-VSCode/v1.2.3 (https://confluent.io; support@confluent.io) sidecar/vunset (Linux/aarch64)"
```

Comment:
These are coming in from the sidecar capitalized, although we're setting os type in there.

Reply:
Right, but we should expect the OS type to be lowercased. See my comment in this review that suggests changes in `SidecarInfo.getUserAgent()`.